apps/src/aiTutor/types.ts

```diff
@@ -1,61 +1,68 @@
 import {
-  AiTutorInteractionSaveStatus,
-  AiTutorTypes,
+  AiTutorInteractionStatus as AITutorInteractionStatus,
+  AiTutorTypes as AITutorTypes,
+  PiiTypes as PII,
 } from '@cdo/apps/util/sharedConstants';
 
-export type ChatCompletionMessage = {
+export type AITutorTypesValue =
+  (typeof AITutorTypes)[keyof typeof AITutorTypes];
+export type AITutorInteractionStatusValue =
+  (typeof AITutorInteractionStatus)[keyof typeof AITutorInteractionStatus];
+
+export {AITutorInteractionStatus, AITutorTypes, PII};
+
+export interface ChatCompletionMessage {
   id: number;
   role: Role;
   chatMessageText: string;
-  status: Status;
+  status: AITutorInteractionStatusValue;
   timestamp?: string;
-};
+}
 
-export type AITutorInteraction = {
+export interface AITutorInteraction {
   userId?: number;
   levelId?: number;
   scriptId?: number;
-  type: TutorType | undefined;
+  type: AITutorTypesValue | undefined;
   isProjectBacked?: boolean;
   prompt: string;
   status: string;
   aiResponse?: string;
-};
+}
 
-export type StudentChatRow = {
+export interface StudentChatRow {
+  aiModelVersion: string;
+  aiResponse?: string;
+  createdAt: string;
   id: number;
-  studentName: string;
-  type: TutorType;
+  levelId?: number;
+  projectId?: string;
   prompt: string;
+  scriptId?: number;
   status: string;
-  aiResponse?: string;
-  createdAt: string;
-};
+  studentName: string;
+  type: AITutorTypesValue;
+  updatedAt?: string;
+  userId: number;
```

Comment:
These fields seemed to be missing, but were on the `chatMessage` on the front-end.